src/hooks/usePatchStatusSelect.ts

```diff
@@ -185,16 +187,15 @@ export const usePatchStatusSelect = (
 };
 
 const reduceBuildVariants = (
+  parentTasksChanged,
   buildVariants,
   patchStatusFilterTerm,
   baseStatusFilterTerm,
   selectedTasks
```

Comment:
Actually could we refactor this to take in an object instead of a bunch of different values. It makes calling it more readable and less error prone when writing it?